data/cookbook/use-basic-auth-on-web-server/00-dream-server.ml

```diff
@@ -0,0 +1,60 @@
+(* 
+   ⚠️  WARNING: DO NOT USE THIS IN PRODUCTION! ⚠️
+   
+   This is a DEMONSTRATION ONLY showing basic HTTP authentication concepts.
+   
+   This code has SERIOUS SECURITY FLAWS:
+   - Passwords stored in plain text
+   - No password hashing
+   - Vulnerable to timing attacks
+   - No rate limiting
+   - No HTTPS enforcement
+   - Improper credential validation
+   
+   For production use:
+   - Consult OWASP Authentication Cheat Sheet
+   - Use established authentication frameworks
```

Comment:
I would add:

Consider using passwordless, phishing-resistant auth like [WebAuthn](https://ocaml.org/p/webauthn) (passkeys).

Reply:
Adding passkey-auth as a recipe to the cookbook sounds like a great idea, in addition.

Having comprehensive docs on stuff like auth is a pretty important signal to people actually building stuff for production.

```suggestion
   - Use established authentication frameworks and schemes (e.g. passwordless, phishing-resistant auth like [WebAuthn](https://ocaml.org/p/webauthn) (passkeys))
```